src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTest.java

```diff
@@ -24,8 +24,15 @@ private void init() {
     new Navigate(driver).toPage(MercurySubpages.MLP_MAIN_PAGE);
   }
 
-  @Test(groups = "mercury_smartBanner_hasFixedPositionOnTopAndCanBeClosed")
-  public void mercury_smartBanner_hasFixedPositionOnTopAndCanBeClosed() {
+  @Test(groups = "mercury_smartBanner_isNotVisibleOnScrollDown")
+  public void mercury_smartBanner_isNotVisibleOnScrollDown() {
+    init();
+
+    smartBanner.scrollDown();
+  }
+
+  @Test(groups = "mercury_smartBanner_canBeClosed")
+  public void mercury_smartBanner_canBeClosed() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)